lib/game.rb

```diff
@@ -2,25 +2,107 @@
 
 class Game
 
-  # def initialize
-  #
-  # end
+  attr_reader :board, :ship
+
+  def initialize
+    @board = Board.new
+    @ship = ship
+    @player_cruiser = Ship.new("Cruiser", 3)
+    @player_submarine = Ship.new("Submarine", 2)
```

Comment:
We will need to add the Player class here and reference ships using player.cruiser/computer.cruiser

Reply:
We can definitely do that! Let's collab to get that done today